src/components/Inputs/Checkbox/Checkbox.vue

```diff
@@ -1,21 +1,23 @@
 <template>
   <div class="input-wrap input-checkbox">
+    <input
+      :id="id"
+      class="is-checkradio is-info"
+      :class="shouldBeChecked ? 'has-background-color' : ''"
+      v-bind="$attrs"
+      :value="value"
+      :checked="shouldBeChecked"
+      :aria-checked="shouldBeChecked"
+      :aria-labelledby="id"
+      :aria-required="$attrs.required"
+      type="checkbox"
+      @change="updateInput"
+      @on="$listeners"
+    >
     <label
       :for="id"
       class="checkbox"
     >
-      <input
-        :id="id"
-        v-bind="$attrs"
-        :value="value"
-        :checked="shouldBeChecked"
-        :aria-checked="shouldBeChecked"
-        :aria-labelledby="id"
-        :aria-required="$attrs.required"
-        type="checkbox"
-        @change="updateInput"
-        @on="$listeners"
-      >
       <slot />
```

Comment:
What are the benefits of using a slot instead of passing a value for the label? Passing a value is cleaner, we don't need a closing tag for the component but maybe there are some benefits of using a slot (like passing html to a label?). I wonder if we could offer both, if no slot use label, or if no label use slot.